### PR TITLE
improve testing harness for liquidation

### PR DESCRIPTION
**Pull request by turadg**
Some CLI bug fixes and additional logging

@JimLarson will revert the verbose logging commit ([cf4c3a8](https://github.com/Agoric/agoric-sdk/pull/7773/commits/cf4c3a8b07b654a9b023726430eee6bd9a1f800f)) before cutting the release.

### CODE REVIEW

File: packages/agoric-cli/src/commands/oracle.js [hunk 1]
```diff
@@ -77,7 +77,7 @@ export const makeOracleCommand = logger => {
 
       /** @type {import('@agoric/smart-wallet/src/offers.js').OfferSpec} */
       const offer = {
-        id: Number(opts.offerId),
+        id: opts.offerId,
```
Comment (dckc) [inline packages/agoric-cli/src/commands/oracle.js:80]: :+1:

File: packages/deployment/upgrade-test/Makefile [hunk 2]
```diff
@@ -9,9 +9,11 @@ endif
 local_sdk:
 	(cd ../ && make docker-build-sdk)
 
+# build main bootstrap
```
Comment (dckc) [inline packages/deployment/upgrade-test/Makefile:12]: seems like there's more to say here while we're at it. But the words aren't coming to me. oh well.

maybe borrow from the `$comment` in the .json config files?

File: packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-10/actions.sh [hunk 3]
```diff
@@ -36,37 +39,37 @@ for i in "${oracles[@]}"; do
     agops oracle accept --offerId "$OFFER_ID" >|"$ORACLE_OFFER"
     agoric wallet print --file "$ORACLE_OFFER"
     agops perf satisfaction --from "$i" --executeOffer "$ORACLE_OFFER" --keyring-backend=test
-    echo "${i}_ORACLE=$OFFER_ID" >> "$HOME/.agoric/envs"
+    echo "${i}_ORACLE=$OFFER_ID" >>"$HOME/.agoric/envs"
```
Comment (dckc) [inline packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-10/actions.sh:42]: double-checking my (lack of) shell-fu: this has no observable runtime impact, right?

File: packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-10/actions.sh [hunk 4]
```diff
@@ -36,37 +39,37 @@ for i in "${oracles[@]}"; do
...
 sleep 65
 
-
-
-
-# ensure params were changed
-test_val "$(agoric follow -l -F  :published.auction.governance -o jsonlines | jq -r .current.ClockStep.value.relValue)" "$CLOCK_STEP"
-test_val "$(agoric follow -l -F  :published.auction.governance -o jsonlines | jq -r .current.StartFrequency.value.relValue)" "$START_FREQUENCY"
+echo ACTIONS ensuring params were changed
+test_val "$(agoric follow -l -F :published.auction.governance -o jsonlines | jq -r .current.ClockStep.value.relValue)" "$CLOCK_STEP"
```
Comment (dckc) [inline packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-10/actions.sh:68]: whitespace change only? seems to be.

File: packages/deployment/upgrade-test/upgrade-test-scripts/env_setup.sh [hunk 5]
```diff
@@ -201,29 +201,38 @@ printKeys() {
...
+  lastStartedBy=$(jq .startedBy "$HOME/.agoric/latestRound-ATOM.json")
+  echo lastStartedBy $lastStartedBy
+  nextOracle="ERROR"
+  # cycle to next among oracles (first of the two governance accounts)
+  case $lastStartedBy in
+    "$GOV1ADDR") nextOracle=$GOV2ADDR;;
+    "$GOV2ADDR") nextOracle=$GOV1ADDR;;
+    *)
+      echo last price was pushed by a different account, using GOV1
+      nextOracle=$GOV2ADDR
```
Comment (dckc) [inline packages/deployment/upgrade-test/upgrade-test-scripts/env_setup.sh]: GOV1 or GOV2?

File: packages/deployment/upgrade-test/upgrade-test-scripts/env_setup.sh [hunk 6]
```diff
@@ -201,29 +201,38 @@ printKeys() {
 echo ENV_SETUP finished
 
 pushPrice () {
+  echo ACTIONS pushPrice $1
   newPrice="${1:-10.00}"
-  for oracleNum in {1..2}; do
-    if [[ ! -e "$HOME/.agoric/lastOracle" ]]; then
-      echo "$GOV1ADDR" > "$HOME/.agoric/lastOracle"
-    fi
+  agoric follow -lF :published.priceFeed.ATOM-USD_price_feed.latestRound -ojson > "$HOME/.agoric/latestRound-ATOM.json"
```
Comment (dckc) [inline packages/deployment/upgrade-test/upgrade-test-scripts/env_setup.sh]: :+1: this seems more reliable

File: packages/inter-protocol/src/auction/auctionBook.js [hunk 7]
```diff
@@ -50,7 +50,7 @@ const DEFAULT_DECIMALS = 9;
  * added in the appropriate place and settled when the price reaches that level.
  */
 
-const trace = makeTracer('AucBook', false);
+const trace = makeTracer('AucBook', true);
```
Comment (dckc) [inline packages/inter-protocol/src/auction/auctionBook.js:53]: did we get the wear-leveling option on these `makeTracer` bits? :)

**Review by dckc — APPROVED** [review]
:+1:

File: packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-10/actions.sh (hunk 3 above)
Comment (turadg) [reply packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-10/actions.sh:42]: right. it was my autoformatter

File: packages/deployment/upgrade-test/upgrade-test-scripts/env_setup.sh (hunk 5 above)
Comment (turadg) [reply packages/deployment/upgrade-test/upgrade-test-scripts/env_setup.sh]: oops. not worth another CI run rn

File: packages/inter-protocol/src/auction/auctionBook.js (hunk 7 above)
Comment (turadg) [reply packages/inter-protocol/src/auction/auctionBook.js:53]: yeah it was part of the durability upgrade

**Review by arirubinstein — APPROVED** [review]

**Comment by arirubinstein** [comment]
the agoric follow -F appears to hang this

```
2023-05-18T05:26:05.315Z block-manager: block 553 commit
2023-05-18T05:26:06.338Z block-manager: block 554 begin
2023-05-18T05:26:06.343Z block-manager: block 554 commit
+++ echo ACTIONS ensure params were changed
ACTIONS ensure params were changed
++++ agoric follow -l -F :published.vaultFactory.managers.manager0.governance -o jsonlines
++++ jq -r .current.DebtLimit.value.value
2023-05-18T05:26:07.352Z block-manager: block 555 begin
2023-05-18T05:26:07.357Z block-manager: block 555 commit
2023-05-18T05:26:08.361Z block-manager: block 556 begin
2023-05-18T05:26:08.366Z block-manager: block 556 commit
+++ test_val 123000000000000 123000000000000
SUCCESS: TEST: unnamedtest: wanted 123000000000000, got 123000000000000
+++ want=123000000000000
+++ got=123000000000000
+++ testname=unnamedtest
+++ [[ 123000000000000 != \1\2\3\0\0\0\0\0\0\0\0\0\0\0\0 ]]
+++ success 'TEST: unnamedtest: wanted 123000000000000, got 123000000000000'
+++ echo 'SUCCESS: TEST: unnamedtest: wanted 123000000000000, got 123000000000000'
+++ pushPrice 12.01
+++ echo ACTIONS pushPrice 12.01
+++ newPrice=12.01
ACTIONS pushPrice 12.01
+++ agoric follow -lF :published.priceFeed.ATOM-USD_price_feed.latestRound -ojson
2023-05-18T05:26:09.371Z block-manager: block 557 begin
2023-05-18T05:26:09.376Z block-manager: block 557 commit
2023-05-18T05:26:10.374Z block-manager: block 558 begin
2023-05-18T05:26:10.378Z block-manager: block 558 commit
2023-05-18T05:26:11.382Z block-manager: block 559 begin
2023-05-18T05:26:11.387Z block-manager: block 559 commit
2023-05-18T05:26:12.389Z block-manager: block 560 begin
2023-05-18T05:26:12.392Z block-manager: block 560 commit
2023-05-18T05:26:13.396Z block-manager: block 561 begin
2023-05-18T05:26:13.400Z block-manager: block 561 commit
2023-05-18T05:26:14.410Z block-manager: block 562 begin
```

**Comment by michaelfig** [comment]
> the agoric follow -F appears to hang this

I've attempted to address this.  We'll see how it goes in CI.

**Pull request merged**